src/components/RenderRocket.js

```diff
@@ -1,24 +1,35 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { reserveRocket } from '../redux/rockets/rockets';
 
 const RenderRocket = (
   {
     name, desc, id, img,
   },
-) => (
-  <li className="rocket-li">
-    <div className="img-container">
-      <img alt={name} src={img} />
-    </div>
-    <div className="rocket-info">
-      <h2>{name}</h2>
-      <p>{desc}</p>
-      <button type="button" id={id}>
-        Reserve rocket
-      </button>
-    </div>
-  </li>
-);
+) => {
+  const dispatch = useDispatch();
+
+  const toggleReserve = (event) => {
```

Comment:
In this case, I checked the function and you are only setting reserved to true, so calling it "toggle reserve" could be confusing for someone who is not familiar with your code;
Is a good practice in React to call actions like "handleEvent" in most cases to avoid this problem, so kindly consider thinking the most propper name, or  calling it "handleClick"